python/nano/src/bigdl/nano/deps/ray/ray_distributed.py

```diff
@@ -45,7 +45,8 @@
 from ray.util.sgd.utils import find_free_port
 from torch.nn import Module
 
-from .environments.ray_envbase import RayEnvironment
+from bigdl.nano.deps.ray.ray_envbase import RayEnvironment
+from ray_api import create_ray_envbase
```

Comment:
I think you should not call `ray_api` inside of `deps/ray/` to avoid circular dependency.

Reply:
OK.